frontend/src/pages/Products/components/ProductCard.tsx

```diff
@@ -0,0 +1,25 @@
+import { Card, CardContent, CardMedia, Typography } from "@mui/material";
+import productStyles from "./styles";
+import { ProductCardProps } from "../../../types/products";
+
+const ProductCard: React.FC<ProductCardProps> = ({ product }) => {
```

Comment:
Handle the case where product might be null or undefined

Reply:
done!